tools/FileTestDefs.sh

```diff
@@ -298,6 +298,10 @@ license() {
               'docs/config/header.html' \
               'docs/config/layout.xml' \
               'docs/config/MathJax.js$' \
+              'external/pybind11/tools/FindPythonLibsNew.cmake' \
```

Comment:
Can you try if the regex `'external/*` works?

Reply:
looks like it does 👍